### DISCUSSION TITLE
TCAL for LTC Input

### DISCUSSION THREAD

**Discussion by RikshaDriver** [gopro/labs#1087] [Q&A]
Was trying to calibrate timecode on H13 and H11 via LTC input and observed some frame differences.

For example, H13 is ahead by 3 frames @ 25fps.

I tried to correct the difference via TCAL=-120 but it doesn't seem to apply the changes. Is TCAL purely for QR codes?

**Comment by dnewman-gpsw** [comment]
TCAL is for at code time, you want the new OLTC on HERO13

**Reply by RikshaDriver** [reply]
Thanks. Just for my understanding, does TCAL get ignored when LTC is used?

Will OLTC be back-ported to the H11 whenever the next Labs release happens?

**Comment by dnewman-gpsw** [comment]
TCAL is the difference between system time a timecode.  When using LTC, the offset can be different, so it has its own.  Back porting is only a maybe for HERO11 (nothing scheduled yet).

**Reply by RikshaDriver** [reply]
Wondering if RLTC and OLTC were implemented in the latest H11 update?

**Reply by dnewman-gpsw** [reply]
Should be, although untested on the H11

**Reply by RikshaDriver** [reply]
I can confirm both OLTC and RLTC are working on the H11.

Interestingly, enabling AUDS also effectively acts as RLTC=1

**Reply by dnewman-gpsw** [reply]
Yes, they both enabled the same audio pipeline.
